Packs/PaloAltoNetworks_IoT3rdParty/pack_metadata.json

```
@@ -0,0 +1,35 @@
+{
+    "name": "Palo Alto Networks IoT 3rd Party Integrations",
+    "description": "Palo Alto Networks IoT 3rd Party Integrations",
+    "support": "community",
+    "currentVersion": "6.0.0",
+    "author": "ZingboxTeam",
```

customers see us as 1 unit, so let's have that as xsoar as well.
```suggestion
    "author": "Cortex XSOAR",
```